pom.xml

```diff
@@ -51,6 +51,13 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>RELEASE</version>
```

Comment:
```
Some problems were encountered while building the effective model for com.example:survey:jar:0.0.1-SNAPSHOT 'dependencies.dependency.version' for junit:junit:jar is either LATEST or RELEASE (both of them are being deprecated) @ line 58, column 13
It is highly recommended to fix these problems because they threaten the stability of your build.
For this reason, future Maven versions might no longer support building such malformed projects.
```

```suggestion
			<version>4.13</version>
```